src/release/process.md

```diff
@@ -84,78 +115,55 @@ Send a PR to the master branch to:
 
 ## Release day (Thursday)
 
-Decide on a time to do the release, T.
-
-- **T-50m** - Run the following command in a shell with [AWS
-  credentials][awscli] in the [simpleinfra] repository:
-
-  ```
-  ./release-scripts/promote-release.py release prod stable
-  ```
-
-  That'll, in the background, schedule the `promote-release` binary to run on
-  the production secrets (not the dev secrets). That'll sign everything, upload
-  it, update the html index pages, and invalidate the CDN. Note that this takes
-  about 30 minutes right now. This will also push a signed tag to rust-lang/rust.
+Decide on a time to do the release. Let Mara know of the time, so that she can
+be ready to post the release on the project's social media channels.
```

Comment:
```suggestion
Decide on a time to do the release. Let the Social Media coordinator (currently Mara)
know of the time, so that she can be ready to post the release on the project's social
media channels.
```